src/backends/interflop-vprec/interflop_vprec.c

```diff
@@ -273,47 +290,97 @@ void _set_vprec_inst_mode(vprec_inst_mode mode) {
 // Round the float with the given precision
 static float _vprec_round_binary32(float a, char is_input, void *context,
                                    int binary32_range, int binary32_precision) {
+  t_context *currentContext = (t_context *)context;
+
+  /* test if a is special cases */
   if (!isfinite(a)) {
     return a;
   }
 
-  /* round to zero or set to infinity if underflow or overflow compare to
+  /* round to zero or set to infinity if underflow or overflow compared to
    * VPRECLIB_BINARY32_RANGE */
   int emax = (1 << (binary32_range - 1)) - 1;
   int emin = (emax > 1) ? 1 - emax : -1;
 
+  /* in absolute error mode, the error threshold also gives the possible
+   * underflow limit */
+  if ((currentContext->relErr == true) && (currentContext->absErr == true)) {
```

Comment:
As in MCA, I'm not use of when to use rel and abs modes at the same time. If we drop the combined mode, this can be simplified.